cmd/contrib/main.go

```diff
@@ -414,6 +429,66 @@ func Try{{ .PlatformNameUpperCamelCase }}ConfigFile() sdk.Importer {
 `,
 }
 
+var credentialTestTemplate = Template{
+	Filename: "{{ .CredentialNameSnakeCase }}_test.go",
+	Contents: `package {{ .Name }}
+
+import (
+	"testing"
+	
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+	
+func Test{{ .CredentialNameUpperCamelCase }}Provisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, {{ .CredentialNameUpperCamelCase }}().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{ // TODO: Check if this is correct
+				fieldname.{{ .FieldName }}: "{{ .TestCredentialExample }}",
+			},
+			ExpectedOutput: sdk.ProvisionOutput{
+				Environment: map[string]string{
+					"{{ .CredentialEnvVarName }}": "{{ .TestCredentialExample }}",
+				},
+			},
+		},
+	})
+}
+
+func Test{{ .CredentialNameUpperCamelCase }}Importer(t *testing.T) {
+	plugintest.TestImporter(t, {{ .CredentialNameUpperCamelCase }}().Importer, map[string]plugintest.ImportCase{
+		"environment": {
+			Environment: map[string]string{ // TODO: Check if this is correct
+				"{{ .CredentialEnvVarName }}": "{{ .TestCredentialExample }}",
+			},
+			ExpectedCandidates: []sdk.ImportCandidate{
+				{
+					Fields: map[sdk.FieldName]string{
+						fieldname.{{ .FieldName }}: "{{ .TestCredentialExample }}",
+					},
+				},
+			},
+		},
+		// TODO: If you implemented a config file importer, add a test file example in {{ .Name }}/fixtures
```

Comment:
I think the best way to go here is to instruct the user to add their own fixture, rather than adding a default one that just never gets used. If the user doesn't add a file importer, it's another responsibility for them to delete it.

Any thoughts?

Reply:
Yeah, I think so too. We could consider to already create the `test-fixtures` dir, though.

Reply:
I think `git` only indexes files, not directories. We could add a `.keep` file to it, but then we might as well add the fixture directly.

I propose we omit both the file and the directory.